.env.example

```diff
@@ -1,3 +1,3 @@
 GITHUB_TOKEN=your_github_token_here
 OPENAI_API_KEY=your_openai_api_key_here
-ASSISTANT_MODEL=openai-gpt-4
```

Comment:
The environment variable ASSISTANT_MODEL has been removed; ensure that its intended use is covered elsewhere.